src/App.tsx

```diff
@@ -15,71 +17,94 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  HandlerSelectedGoods = () => {
+    this.setState({ selectedGood: '' });
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  HandlerRemoveGood = () => {
+    this.setState({ selectedGood: '' });
```

Comment:
Same here. And what's the difference between these two functions, if both of them do the same?
They are just resetting the selectedGood